tests/actions/activeOrderAction.test.js

```diff
@@ -0,0 +1,48 @@
+import expect from 'expect';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
```

Comment:
All these have already been declared globally, please refer to the setup.js file to avoid importing locally again